<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>My Resume</title>
</head>
<body>
    <h1>Eudean Shaw</h1>
    <img src="./Me.PNG" alt="picture of Eudean Shaw" />
    <h2>Summary</h2>
    <h3>Leads teams and projects to develop customized IT solutions</h3>
    Detail-focused architect, application development leader delivering automated applications
    that align with unique and demanding initiatives in the biotechnology space. Trusted advisor to leadership teams
    in identifying requirements and recommending high-quality IT solutions, adhering to strict standards, data
    integrity, and protocols.
    <hr />
    <h2>Education</h2>
    <h3>
        Doctorate of Philosophy (PhD), Biology, Massachusetts Institute of Technology, Cambridge, MA
    </h3>
    <ul>
        <li>
            Dissertation: The Necessity of Debranching in Starch Biosynthesis. Advisor: Dr. Anthony J. Sinskey
        </li>
    </ul>
    
    <h3>
        Bachelor of Arts (BA), Phi Beta Kappa, Biochemistry and Molecular Biology, University of California,
        Berkeley, Berkeley, CA
    </h3>
    <ul>
        <li>
            Honors Thesis: Structure / Function Analysis of the Periplasmic Histidine-Binding 
            Protein from Salmonella typhimurium. Advisor: Dr. Giovanna F.-L. Ames
        </li>
    </ul>
    <hr />
    <h2>Work Experience</h2>
    <h3>
        LILLY BIOTECHNOLOGY CENTER OF ELI LILLY, San Diego, CA
    </h3>	
    <h4>Senior Advisor	2022 - present</h4>

    Improved process for debugging and testing Biologica. Created custom solutions for acquiring and analyzing
    Carterra and micro drop data. Created an antigen registration system. Creating a cell line registration
    system. Implementing a visual block editor for development and upload of experimental protocols.

    <h4>Senior Scientist 	2017 - 2022</h4>
    Worked on the Biologica project - an end-to-end enterprise platform for biologics drug discovery, which
    includes an electronic lab notebook, molecular biology tools, inventory management, data capture, data analysis,
    and predictive pipelines.
    <ul>
        <li>
            Engaged stakeholders in requirements gathering and biweekly demos to solicit feedback.
        </li>
        <li>
            Coached and motivated developers (in San Diego, Indianapolis, and India) to write maintainable code and
            follow best practices.
        </li>
        <li>
            Saved $400K by writing C++ code myself to implement custom data analysis and visualization.
        </li>
    </ul>
 
    <h3>
        JANSSEN PHARMACEUTICAL COMPANIES OF JOHNSON & JOHNSON (PREVIOUSLY EGEA BIOSCIENCES), San Diego, CA
    </h3>
    <h4>Senior Scientist	2004 - 2017</h4>
    Promoted to work cross-functionally with a team of 200 scientists to develop customized applications in support
    of bio-pharmaceutical research. Complied with strict quality standards, data integrity guidelines, and
    laboratory protocol. 
    <ul>
        <li>
            Developed a suite of customized applications to automate gene assembly technology.
        </li>
        <li>
            Automated transfer of data from 15 different lab instruments to OpenLab by working with key
            stakeholders to gather requirements and recommend optimal solutions.
        </li>
        <li>
            Replaced Prism / Excel with enhanced application for improving data analysis, visualization, and
            reporting.
        </li> 
        <li>Developed lab instrument management system (LIMS) for sequencing facility, troubleshooting and
            resolving complex issues with minimal disruption to work processes.
        </li>
        <li>Optimized inventory system for research cell lines by designing new functionality to support
            research objectives.
        </li> 
    </ul>
    
    <h4>Research Scientist	2001 - 2004</h4>
    Hired as in-house expert for protein design. Developed custom bioinformatics solutions for sequence analysis
    and processing. Researched protein-ligand interactions through molecular dynamics simulations. Developed and
    maintained software to automate core technology.
    <ul>
        <li>
            Took initiative to identify opportunity and automate system in support of research objectives;
            developed and presented recommendations to leadership team, gaining buy-in and securing funding for
            application development.
        </li>
        <li>
            Applied the algorithm invented in previous position to design over 100 new antibodies with improved
            therapeutic properties.
        </li>
    </ul>
 
    <h3>
        RENSSELAER POLYTECHNIC INSTITUTE, Troy, NY	2000 - 2001
    </h3>
    <h4>Post-Doctoral Research Associate</h4> 
    Conducted research at the interface of chemical engineering and bioinformatics to improve understanding of
    relationships between protein structure and functions.
    <ul>
        <li>
            Developed bioinformatic tools to study α/β hydrolases.
        </li>
        <li>
            Invented and published an algorithm to predict likelihood of protein residues being responsible for
            desirable properties, such as substrate specificity or activity.
        </li>
    </ul>

    <h3>
        BARD COLLEGE, Annandale-on-Hudson, NY	1999
    </h3>
    <h4>Assistant Adjunct Professor</h4> 
    Taught two classes.
    <ul>
        <li>
            Biochemistry: lecture and lab.
        </li>
        <li>
            Bioinformatics: seminar and lab.
        </li>
    </ul>
    <hr />
    <h2>Skills</h2>
    <ul>
        <li>
            C/C++, Qt, C#, .NET, Java, R, Shiny, VBA, Perl, WPF, WinForms, SQL/NoSQL, Objective-C, Python, HTML, CSS,
            JavaScript, Git
        </li>
        <li>
            AWS: DynamoDB, S3, CloudFront, Route53, Lambda, SAM
        </li>
        <li>
            Completed courses in Situational Leadership, Emotional Intelligence, and the Scaled Agile Framework
        </li>
    </ul>
    <hr />
    <h2>Awards</h2>
    <ul>
        <li>
            Eli Lilly CIO Choice Winner - Covid Therapeutic Response, 2021
        </li>
        <li>
            Eli Lilly CIO Award - Biologica, 2019
        </li>
        <li>
            Centocor Vice Presidents' Research Award for Outstanding Technical Achievement, 2007
        </li>
        <li>
            MIT $50K Business Plan Competition, Runner-Up, 1999
        </li>
        <li>
            MIT $1K Business Idea Competition, Winner, 1998
        </li>
        <li>
            NIH Biotechnology Training Grant, 1997 - 1999
        </li>
        <li>
            Howard Hughes Medical Institute, BioSciences Education Initiative Grant, 1993
        </li>
        <li>
            Sigma Xi, The Scientific Research Society, Grant-in-Aid of Research, 1993
        </li>
        <li>
            UC Berkeley, President's Undergraduate Fellowship, 1993
        </li>
        <li>
            UC Berkeley, Chancellor's Scholarship, 1990 - 1994
        </li>
    </ul>
    <hr />
    <h2>Publications</h2>
    <ul>
        <li>
            Phan S, Walmer A, Shaw E, Chai Q. 2022 High-throughput profiling of antibody self-association in multiple
            formulation conditions by PEG stabilized self-interaction nanoparticle spectroscopy. mAbs 14:1.
        </li>
        <li>
            Shaw E, Dordick JS. 2002. Predicting Amino Acid Residues Responsible for Enzyme Specificity Solely from
            Protein Sequences. Biotechnology and Bioengineering 79:295-300.
        </li>
        <li>
            Shaw E, McCue LA, Lawrence CE, Dordick JS. 2002. Identification of a Novel Class in the α/β Hydrolase
            Fold Superfamily: The N-myc Differentiation Related Proteins. Proteins: Structure, Function, and
            Genetics 47:163-168.
        </li>
        <li>
            Dauvillee D, Colleoni C, Shaw E, Mouille G, D'Hulst C, Morell M, Samuel MS, Bouchet B, Gallant DJ,
            Sinskey A, Ball S. 1999. Novel, Starch-Like Polysaccharides Are Synthesized by an Unbound Form of
            Granule-Bound Starch Synthase in Glycogen-Accumulating Mutants of Chlamydomonas reinhardtii. Plant
            Physiology 119:321-329.
        </li>
        <li>
            Wolf A, Shaw EW, Oh B-H, Bondt HD, Joshi AK, Ames GF-L. 1995. Structure / Function Analysis of the
            Periplasmic Histidine-Binding Protein. The Journal of Biological Chemistry 270:16097-16106.
        </li>
        <li>
            Wolf A, Shaw EW, Nikaido K, Ames GF-L. 1994. The Histidine-Binding Protein Undergoes Conformational
            Changes in the Absence of Ligand as Analyzed with Conformation-Specific Monoclonal Antibodies. The
            Journal of Biological Chemistry 269:23051-23058.
        </li>
    </ul>
    <hr />
    <h2>Presentations</h2>
    <ul>
        <li>
            Introducing the Assay Dev Toolkit. Society for Laboratory Automation and Screening International
            Conference, San Diego, CA, January 23-27, 2021.
        </li>
        <li>
            Functional Classification of the PHA Synthase Superfamily. American Chemical Society National Meeting,
            San Diego, CA, April 1-5, 2001.
        </li>
        <li>
            An Algorithm for Identifying Amino Acid Residues Responsible for Substrate Specificity. The Dow
            Chemical Company, San Diego, CA, April 2, 2001.
        </li>
        <li>
            Functional Classification of the PHA Synthase Superfamily. American Institute of Chemical Engineers
            National Meeting, Los Angeles, CA, November 12-17, 2000.
        </li>
        <li>
            The Necessity of Debranching in Starch Biosynthesis. DuPont Experimental Station, Wilmington, DE, June 7,
            1999.
        </li>
        <li>
            Cloning of the STA7 Locus from Chlamydomonas reinhardtii. FAIR European Starch Meeting, Le Croisic,
            France, June 25-27, 1997.
        </li>
    </ul>
    <hr />
    <h2>Other</h2>
    <ul>
        <li>
            <a href="./hobbies.html">My Hobbies</a>
        </li>
        <li>
            <a href="./contact.html">Contact Me</a>
        </li>
    </ul>
    
    
    <footer>
        <p>(c) 2023 Eudean Shaw</p>
    </footer>
</body>
</html>